### PR TITLE
Grammar fixes in README.rst

### PR DESCRIPTION
**Pull request by Vgr255**
What the title says

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,16 +15,15 @@ between the programmer and the raw bytes read by the eval loop so that we can
 more easily inspect and modify bytecode.
 
 ``codetransformer`` is motivated by the need to override parts of the python
-language that not already hooked into through data model methods. For example:
+language that is not already hooked into through data model methods. For example:
```
Comment (llllllllll) [inline README.rst]: I think this should be `are`

**Comment by llllllllll** [comment]
Thank you for reading through this!

File: README.rst (hunk 1 above)
Comment (Vgr255) [inline README.rst]: Indeed, well-spotted.

**Comment by Vgr255** [comment]
No prob! I'm slowly getting back into this thing, so I figured I'd start with the documentation, which is rather easy to get right (and wrong :)

**Comment by llllllllll** [comment]
Do you mind prefixing your commits with `DOC:`? will merge after

**Comment by Vgr255** [comment]
I don't mind, but git is being overly stupid tonight. When it stops trying to be too clever for its own good, I'll update the commit messages and push to the branch.

**Pull request merged**